### DISCUSSION TITLE
手机网络下无法稳定访问

### DISCUSSION THREAD

**Discussion by kajunmiemie** [trojan-gfw/igniter#379] [Suspicious Problems]
在连接wifi情况下打开均能连接，用手机网络情况下，在部分地区可以连接，但到其他地方就会中断连接无法连接上了

**Comment by TchaikovDriver** [comment]
这个偏向于运营商的问题

**Comment by kajunmiemie** [comment]
Describe the bug
在移动网络下服务器出现错误
SSL handshake failed: no shared cipher
在wifi下正常

To Reproduce
和证书签名算法有关。Trojan采用RSA证书正常 ECC证书出现错误
cipher使用默认设定
"cipher": "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384",
"cipher_tls13": "TLS_AES_128_GCM_SHA256:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384",
只要手机处于移动网络下出现。处于wifi模式下错误不会出现 哪怕手机连接的是其他手机放出的热点也不会出现这个错误。

在trojan的issue里找到了这个关于igniter移动网络下的问题
